cfgov/unprocessed/css/organisms/featured-content-module.less

```diff
@@ -0,0 +1,104 @@
+// Duplicated from current implementation in Capital Framework v4.6.0
+// with minor edits (to be released in CFv4.7.0)
+// This can be deleted safely when this project is ported to CFv4
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  // Fix the doubled border issue
+  margin-right: -1px;
+  margin-left: -1px;
+
+  background-color: @block__bg;
+
+  .respond-to-max( @bp-xs-max, {
+    padding-right: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em);
+    padding-left: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em);
+  } );
+
+  .respond-to-range( @bp-sm-min, @fcm-bp-max, {
+    padding-right: unit( @grid_gutter-width / @base-font-size-px, em);
+    padding-left: unit( @grid_gutter-width / @base-font-size-px, em);
+  } );
+
+  &_text {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em);
+    padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em);
+
+    .respond-to-min( @fcm-bp-min, {
+      padding-right: unit( @grid_gutter-width / @base-font-size-px, em);
+      padding-left: unit( @grid_gutter-width / @base-font-size-px, em);
+      width: 300px; // width of 5 of 12 cols at 799px - @grid_gutter-width padding
+
+      float: left;
+    } );
+  }
+
+  &_visual {
+    .respond-to-min( @fcm-bp-min, {
+      overflow: hidden;
+
+      .lt-ie10 & {
+        position: absolute;
+        top: 0;
+        right: 0%;
```

Comment:
This could be unit-less, right? `right: 0;`. I think that's what a CSS linter would recommend.